docs/sources/write/style-guide/inclusive-writing/index.md

```diff
@@ -95,3 +95,14 @@ Use the following approach to describe relationships between nodes or processes:
 - Use _secondary_, _replica_, or _child_, instead of _slave_.
 
 Avoid _master_ or _slave_.
+
+### Right and left
+
+Use the terms _right_ and _left_ with a qualifier like _top_ or _lower_ like so:
+
+- top-right
+- bottom-left
+- upper-left
+- lower-right
+
+Avoid using _right_ and _left_ without a qualifier.
```

Comment:
I think the last line about not using these words without a qualifier is redundant and messes with the flow if I add this bit about what qualifiers to not use. 
```suggestion

Don't use the word _hand_ as a qualifier (for example, _right-hand corner_).
```